token/services/db/dbtest/identity.go

```diff
@@ -16,7 +16,20 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var IdentityCases = []struct {
+func IdentityTest(t *testing.T, cfgProvider cfgProvider) {
```

Comment:
The test cases (IdentityCases here) are not exporter anymore. Only IdentityTest is exported and used by the 3 drivers (memory, postgres, sqlite). The db cases are agnostic to the underlying driver.